spec/views/bids/create.html.erb_spec.rb

```diff
@@ -0,0 +1,5 @@
+require 'rails_helper'
+
+RSpec.describe "bids/create.html.erb", type: :view do
```

Comment:
I don't think we ever write specs for views. These auto-generated specs could be deleted.